package.json

```diff
@@ -70,7 +70,8 @@
     "prepare": "husky install"
   },
   "devDependencies": {
-    "@aws-sdk/credential-providers": "3.53.0",
+    "@aws-sdk/client-s3": "3.450.0",
+    "@aws-sdk/credential-providers": "3.450.0",
```

Comment:
@trivikr this still seems to be a downgrade after you updated this with `yarn`.

Reply:
> this still seems to be a downgrade after you updated this with `yarn`.

I didn't get you. The diff added latest exact versions of `@aws-sdk/client-s3` and `@aws-sdk/credential-providers` at the time of development. What's being downgraded?

Reply:
Nope it's me. I was seeing `53` -> `45` not -> `450`. Sorry... 😅